### ISSUE TITLE
Quickshow crashes on ctrl+c with large gif images

### ISSUE THREAD

**Issue by tomuta** [SerenityOS/serenity#3417]
```
[QuickShow(61:61)]: CPU[0] NP(error) fault at invalid address V0x06b3b000
[QuickShow(61:61)]: Unrecoverable page fault, read from address V0x06b3b000
[QuickShow(61:61)]: CRASH: CPU #0 Page Fault. Ring 3.
[QuickShow(61:61)]: exception code: 0004 (isr: 0000
[QuickShow(61:61)]:   pc=001b:08051d87 flags=0202
[QuickShow(61:61)]:  stk=0023:01a02750
[QuickShow(61:61)]:   ds=0023 es=0023 fs=0023 gs=002b
[QuickShow(61:61)]: eax=046b1000 ebx=01a027d8 ecx=00000001 edx=046b1000
[QuickShow(61:61)]: ebp=01a027b8 esp=c35b1fe8 esi=06b3b000 edi=04f7b000
[QuickShow(61:61)]: cr0=80010013 cr2=06b3b000 cr3=00ab7000 cr4=00300ee4
[QuickShow(61:61)]: code: f3 a4 eb 03 c6 00 00 8b
[QuickShow(61:61)]: 0x08051d87  GUI::Clipboard::set_data(AK::Span<unsigned char const>, AK::String const&, AK::HashMap<AK::String, AK::String, AK::Traits<AK::String> > const&) +69

[QuickShow(61:61)]: 0xc01631bb  Kernel::Process::crash(int, unsigned int, bool) +1041
[QuickShow(61:61)]: 0xc0120b21  illegal_instruction_handler +0
[QuickShow(61:61)]: 0xc012215b  page_fault_handler +972
[QuickShow(61:61)]: 0xc011e413  page_fault_asm_entry +38
[QuickShow(61:61)]: 0x08052031  GUI::Clipboard::set_bitmap(Gfx::Bitmap const&) +403
[QuickShow(61:61)]: 0x0804ebd6  GUI::Action::activate(Core::Object*) +172
[QuickShow(61:61)]: 0x08082810  GUI::WindowServerConnection::handle(Messages::WindowClient::KeyDown const&) +238
[QuickShow(61:61)]: 0x08082db1  WindowClientEndpoint::handle(IPC::Message const&) +175
[QuickShow(61:61)]: 0x0805dc6e  IPC::ServerConnection<WindowClientEndpoint, WindowServerEndpoint>::handle_messages() +106
[QuickShow(61:61)]: 0x080c3f70  Core::Object::dispatch_event(Core::Event&, Core::Object*) +74
[QuickShow(61:61)]: 0x080bce0a  Core::EventLoop::pump(Core::EventLoop::WaitMode) +308
[QuickShow(61:61)]: 0x080bcf89  Core::EventLoop::exec() +43
[QuickShow(61:61)]: 0x0804982b  main +5152
[QuickShow(61:61)]: 0x0804a8ee  _start +94
```

1. Download a large gif file, e.g. from http://www.lunarpup.com/animations/animations.html
2. Open it in QuickShow, should render ok
3. Press Ctrl+C and observe QuickShow crashing

**Comment by alimpfard** [comment]
Probably fixed by 02f251b?

**Comment by tomuta** [comment]
> Probably fixed by [02f251b](https://github.com/SerenityOS/serenity/commit/02f251bb4af00d6d145d91f83f77f24d4da253fa)?

Man, you are fast! Yes, indeed, that fixes it :-)

**Comment by alimpfard** [comment]
More like kling is fast at ninja-merging PRs lol
